improver/blending/spatial_weights.py

```diff
@@ -216,51 +216,6 @@ def multiply_weights(weights_from_mask, one_dimensional_weights_cube, blend_coor
         result = result.merge_cube()
         return result
 
-    @staticmethod
-    def normalised_masked_weights(weights_cube, blend_coord):
```

Comment:
There are some docstrings that now need updating to say the weights returned from this plugin aren't normalised